pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mergify_cli"
-version = "0.0.0"
+version = "2023.09.27.08.14"
```

Comment:
is there a meaning?

Reply:
It's the current version number, which is the date of release.

Reply:
Is it date + time? That's the first time I see this :|

Reply:
Yeah, it's the date and the time (hours and minutes).
It was introduced here #249, specified by [this task](https://linear.app/mergify/issue/MRGFY-2664/fix-mergify-cli-pypi-release). We probably discussed the version system orally or on Slack. I don't find any track of it.